resteasy/src/test/java/com/cerner/beadledom/resteasy/TestProviderWithBindingAnnotation.java

```diff
@@ -0,0 +1,29 @@
+package com.cerner.beadledom.resteasy;
+
+import com.cerner.beadledom.guice.BindingAnnotations;
+
+import javax.inject.Provider;
+import java.lang.annotation.Annotation;
+
+/**
+ * Provides a fully-constructed instance of {@link TestJaxRsProvider} by taking in a binding annotation.
+ */
+public class TestProviderWithBindingAnnotation implements Provider<TestJaxRsProvider> {
+
+  /**
+   * Constructor for TestProviderWithBindingAnnotation.
```

Comment:
This comment isn't helpful. If an author cannot tell that this is a constructor they shouldn't be messing with it.

Reply:
https://github.com/cerner/beadledom/pull/143/commits/75d364f068be5b48f12a864aa2a8bd938fb929b8